CMakeLists.txt

```diff
@@ -1,22 +1,65 @@
+cmake_minimum_required(VERSION 3.22)
```

Comment:
This is very new. Is there something you've done that actually requires this? For a couple points of reference (not that Ubuntu is the only relevant distro): Ubuntu 18.04 and 20.04 (both LTS) distribute CMake 3.10 and 3.18, respectively. DFHack claims to require 2.8.12 at a minimum, although I suspect the true minimum is a bit newer.

Reply:
No idea, I've just been using cmake since this version so I have no other baseline.